Waffle october 3

Thinking-waffle opened the pull request:
The argeads are back.

It needs the cosmetic tag of the Macedonian empire to be visible (MAE_neutrality, MAE_fascism) and the Bftb greek tree. Check that there is no icon overlap (with continuous focuses for example). All because somebody complained about it on discord. "Things I do for love!"

Also more names, callsigns and operation names.

Thinking-waffle committed:
Tuvan focus icon.

Thinking-waffle committed:
more operation call signs and family names

thanks chatGPT

Thinking-waffle committed:
Argead's rebirth, surprise.

Let's ignore all the migrations of Turks, Islam and all that fuss, the Argeads through the house of Denmark-Greece.

Just check that the icons don't overlap with continuous focuses or something else (I tried to check) .

Thinking-waffle committed:
Update r56_countries_cosmetic_l_english.yml

Thinking-waffle committed:
fixing Pu Yi field marshal

marijn211 commented:
The legacy ID you remove seems to be referred to in R56_on_actions (and unique commander IDs)

Thinking-waffle committed:
Merge branch 'Beta-Edits' into waffle-october-3

Thinking-waffle committed:
Update r56_arctic_minor_military_shared.txt

making the samis OP

Thinking-waffle committed:
Update r56_arctic_minor_military_shared.txt

Thinking-waffle committed:
mostly cold icons

Thinking-waffle committed:
text of the new filters

Thinking-waffle committed:
Update r56_finland.txt

Thinking-waffle committed:
fixing ski troops in r56 Finnish and Norwegian trees

marijn211 committed:
Removed Pu Yi legacy ID references

Thinking-waffle committed:
Your wholesome sami portrait merge

-added new generic sami and Finnish portraits to r56_portraits rather than 00_portraits to ensure having to handle two files for no reason.

Added finnish, sami, karelian portrait paths to interface/_leader_portraits.gfx as those are free and shouldn't cause dlc complications.

Implementing some of those portraits through the Finnish character list is most probably still required.

"generic versions" of those new Finnish portraits could be useful to enrich the generic portrait pool with limited effort... and make it more relevant for the Samis.

Thinking-waffle committed:
Sami councils are OP pls nerf

Added traits to the Sami councils

Added fascist coup mechanic to remove the pesky council and get personal rule.of an army commander.

Thinking-waffle committed:
Update SMI_l_english.yml

marijn211 committed:
Merge branch 'Beta-Edits' into waffle-october-3

marijn211 committed:
Missing loc

marijn211 committed:
Fixed incorrect GFX reference from vanilla

marijn211 committed:
Update norway.txt

marijn211 committed:
Small Sweden fix
